Namespace(arch='resnet50', batch_size=16, bottleneck_dim=256, data='ImageNet50', epochs=20, iters_per_epoch=1000, log='dann', lr=0.01, lr_decay=0.75, lr_gamma=0.001, momentum=0.9, no_hflip=False, no_pool=False, norm_mean=(0.485, 0.456, 0.406), norm_std=(0.229, 0.224, 0.225), per_class_eval=False, phase='test', print_freq=100, resize_size=224, root='/media/ImageNet_dataset/v100/jin/', scratch=False, seed=None, source=['IN'], target=['IN-test'], trade_off=1.0, train_resizing='custom.target', val_resizing='custom.target', weight_decay=0.001, workers=2)
train_transform:  Compose(
    Compose(
    ResizeImage(size=(224, 224))
    CenterCrop(size=(224, 224))
)
    RandomHorizontalFlip(p=0.5)
    ToTensor()
    Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225))
)
val_transform:  Compose(
    Compose(
    ResizeImage(size=(224, 224))
    CenterCrop(size=(224, 224))
)
    ToTensor()
    Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225))
)
=> using model 'resnet50'
Traceback (most recent call last):
  File "dann.py", line 290, in <module>
    main(args)
  File "dann.py", line 116, in main
    checkpoint = torch.load(logger.get_checkpoint_path('best'), map_location='cpu')
  File "/home/jin2/.local/lib/python3.8/site-packages/torch/serialization.py", line 527, in load
    with _open_zipfile_reader(f) as opened_zipfile:
  File "/home/jin2/.local/lib/python3.8/site-packages/torch/serialization.py", line 224, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: version_ <= kMaxSupportedFileFormatVersion INTERNAL ASSERT FAILED at /pytorch/caffe2/serialize/inline_container.cc:132, please report a bug to PyTorch. Attempted to read a PyTorch file with version 3, but the maximum supported version for reading is 2. Your PyTorch installation may be too old. (init at /pytorch/caffe2/serialize/inline_container.cc:132)
frame #0: c10::Error::Error(c10::SourceLocation, std::string const&) + 0x33 (0x7f6c66827193 in /home/jin2/.local/lib/python3.8/site-packages/torch/lib/libc10.so)
frame #1: caffe2::serialize::PyTorchStreamReader::init() + 0x1f5b (0x7f6bd664fafb in /home/jin2/.local/lib/python3.8/site-packages/torch/lib/libtorch.so)
frame #2: caffe2::serialize::PyTorchStreamReader::PyTorchStreamReader(std::string const&) + 0x64 (0x7f6bd6650d14 in /home/jin2/.local/lib/python3.8/site-packages/torch/lib/libtorch.so)
frame #3: <unknown function> + 0x6c1c56 (0x7f6c678f4c56 in /home/jin2/.local/lib/python3.8/site-packages/torch/lib/libtorch_python.so)
frame #4: <unknown function> + 0x295688 (0x7f6c674c8688 in /home/jin2/.local/lib/python3.8/site-packages/torch/lib/libtorch_python.so)
frame #5: PyCFunction_Call + 0x56 (0x5eda96 in /usr/bin/python3.8)
frame #6: _PyObject_MakeTpCall + 0x28f (0x5ee15f in /usr/bin/python3.8)
frame #7: /usr/bin/python3.8() [0x4e670d]
frame #8: PyVectorcall_Call + 0x1ef (0x5ed72f in /usr/bin/python3.8)
frame #9: /usr/bin/python3.8() [0x59ee6d]
frame #10: _PyObject_MakeTpCall + 0x1ed (0x5ee0bd in /usr/bin/python3.8)
frame #11: _PyEval_EvalFrameDefault + 0x617d (0x56a51d in /usr/bin/python3.8)
frame #12: _PyEval_EvalCodeWithName + 0x23d (0x561ffd in /usr/bin/python3.8)
frame #13: _PyFunction_Vectorcall + 0x247 (0x5f0dd7 in /usr/bin/python3.8)
frame #14: /usr/bin/python3.8() [0x59e949]
frame #15: _PyObject_MakeTpCall + 0x1ed (0x5ee0bd in /usr/bin/python3.8)
frame #16: _PyEval_EvalFrameDefault + 0x5b33 (0x569ed3 in /usr/bin/python3.8)
frame #17: _PyEval_EvalCodeWithName + 0x23d (0x561ffd in /usr/bin/python3.8)
frame #18: _PyFunction_Vectorcall + 0x442 (0x5f0fd2 in /usr/bin/python3.8)
frame #19: _PyEval_EvalFrameDefault + 0x19c5 (0x565d65 in /usr/bin/python3.8)
frame #20: _PyEval_EvalCodeWithName + 0x23d (0x561ffd in /usr/bin/python3.8)
frame #21: _PyFunction_Vectorcall + 0x247 (0x5f0dd7 in /usr/bin/python3.8)
frame #22: _PyEval_EvalFrameDefault + 0x629 (0x5649c9 in /usr/bin/python3.8)
frame #23: _PyEval_EvalCodeWithName + 0x23d (0x561ffd in /usr/bin/python3.8)
frame #24: /usr/bin/python3.8() [0x65bbfe]
frame #25: PyRun_FileExFlags + 0x97 (0x65bcd7 in /usr/bin/python3.8)
frame #26: PyRun_SimpleFileExFlags + 0x17f (0x65cb2f in /usr/bin/python3.8)
frame #27: Py_RunMain + 0x20e (0x68470e in /usr/bin/python3.8)
frame #28: Py_BytesMain + 0x29 (0x684a99 in /usr/bin/python3.8)
frame #29: __libc_start_main + 0xe7 (0x7f6c6e695bf7 in /lib/x86_64-linux-gnu/libc.so.6)
frame #30: _start + 0x2a (0x5f5c0a in /usr/bin/python3.8)

